### PR TITLE
chore: codecov badge fix and readme cleanup

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
-# Outline Apps
+# <img alt="Outline Manager Logo" src="docs/resources/logo_manager.png" title="Outline Manager" width="32">&nbsp;&nbsp;Outline Apps&nbsp;&nbsp;<img alt="Outline Client Logo" src="docs/resources/logo_client.png" title="Outline Client" width="32">
```
Comment (jyyi1) [inline README.md:1]: *nit* Maybe we can put both icons before the text?

File: client/README.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
-# Outline Client
+# <img alt="Outline Client Logo" src="../docs/resources/logo_client.png" title="Outline Client" width="32">&nbsp;&nbsp;Outline Client
 
-![Build and Test](https://github.com/Jigsaw-Code/outline-apps/actions/workflows/build_and_test_debug_client.yml/badge.svg?branch=master) [![codecov](https://codecov.io/gh/Jigsaw-Code/outline-apps/graph/badge.svg?token=gasD8v5tjn)](https://codecov.io/gh/Jigsaw-Code/outline-apps)
-
-> [!NOTE]
-> Test coverage currently only tracks the Apple Libraries and core web view code.
+![Build and Test](https://github.com/Jigsaw-Code/outline-apps/actions/workflows/build_and_test_debug_client.yml/badge.svg?branch=master) 
 
 The Outline Client is a cross-platform VPN or proxy client for Windows, macOS, iOS, Android, and ChromeOS. The Outline Client is designed for use with the [Outline Server](https://github.com/Jigsaw-Code/outline-server) software, but it is fully compatible with any [Shadowsocks](https://shadowsocks.org/) server.
```
Comment (jyyi1) [inline client/README.md:5]: Not your change, but "ChromeOS"?

Comment (fortuna) [reply client/README.md]: The Android app runs on ChromeOS. At least it used to. I remember we had to do some change to make it work there.

File: client/README.md [hunk 3]
```diff
@@ -1,11 +1,8 @@
-# Outline Client
+# <img alt="Outline Client Logo" src="../docs/resources/logo_client.png" title="Outline Client" width="32">&nbsp;&nbsp;Outline Client
```
Comment (fortuna) [inline client/README.md:1]: I like the icons!

**Review by fortuna — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
 
-## Access to the free and open internet!
+> [!WARNING]
+> Codecov currently only tracks the `/client` Apple libraries and core web view code.
 
-Outline makes it easy to create a VPN server, giving anyone access to the free and open internet.
+Outline makes it easy for anyone to create a VPN server, allowing you to share access to the free and open internet with those in need. **We have two core applications:**
 
-We have two core apps
+&nbsp;&nbsp;&nbsp;&nbsp;<img alt="Outline Manager Logo" src="docs/resources/logo_manager.png" title="Outline Manager" width="14">&nbsp;&nbsp;**Outline Manager** ([`/server_manager`](server_manager)): A graphical user interface for managing Outline servers. It is available for Windows, macOS, and Linux. [You can install the manager here](https://getoutline.org/get-started/#step-1).
```
Comment (sbruens) [inline README.md:10]: Any reason you removed the bullet/list item? That nicely indents the whole paragraph, versus the awkward hanging indent it now has.

**Pull request merged**